firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -166,18 +165,8 @@ public class FirebaseCrashlytics {
 
     // Kick off actually fetching the settings.
     settingsController
-        .loadSettingsData(threadPoolExecutor)
```

Comment:
Does `threadPoolExecutor` have to be sequential? Can we directly pass `backgroundExecutorService`?